README.md

```diff
@@ -2,14 +2,90 @@
 
 # Wikipedia KaiOS 
 
-The official Wikipedia KaiOS app.
+The official Wikipedia KaiOS app. [KaiOS](https://developer.kaiostech.com/) is a web-based mobile operating system that enables a new category of smart feature phones, a successor of the discontinued Firefox OS.
 
-Tasks and issues are tracked on [Phabricator](https://phabricator.wikimedia.org/project/profile/4305/).
+KaiOS applications are similar to Progressive Web Apps (PWAs):
+* Pure web applications
+* Manifest file
+* Installed on the device
+* Have access to the required APIs for notifications, offline, storage, audio, etc
+
+They are also different:
+* Format of the manifest is different
+* Size of the icons is specific to KaiOS
+* Need to be designed for a small, non-touch screen with a few physical buttons
+* Several APIs are specific to the platform (not web standards)
+* Running performance on the device
+
+The Wikipedia KaiOS app is built with [Preact](https://preactjs.com/) and we use [Cypress](https://www.cypress.io/) for testing. Tasks and issues are tracked on [Phabricator](https://phabricator.wikimedia.org/project/profile/4305/).
```

Comment:
I would prefer to link to the github repos instead of the website, to keep it all inside the open source realm

Reply:
Unfortunately, we need to keep Phabricator link as it is our main issue and tracking place.

Reply:
@jpita do you mean Github links for Preact and Cypress? 

https://github.com/preactjs/preact
https://github.com/cypress-io/cypress

I like that idea much better. And we could keep the Phabricator link then

Reply:
Assuming that's the case and will update it as such, let me know if you meant something different

Reply:
yes, sorry, couldn't comment yesterday